lib/rototiller/task/params/env_var.rb

```diff
@@ -47,16 +47,16 @@ def message
         this_message = String.new
 
         if env_status    == STATUS[:nodefault_noexist]
-          this_message << red_text('ERROR: environment-variable not set and no default provided: ')
+          this_message << red_text(' - ERROR: environment-variable not set and no default provided: ')
```

Comment:
why?  i'm not saying i disagree.  just wondering what the thought is for the hyphen